tests/functional/test_utilities_parameters.py

```diff
@@ -1534,6 +1534,36 @@ def _get_multiple(self, path: str, **kwargs) -> Dict[str, str]:
     assert str_value == json.dumps(mock_body_json)
 
 
+def test_appconf_get_app_config_new(monkeypatch, mock_name, mock_value):
+    """
+    Test get_app_config() without a default provider
+    """
```

Comment:
```suggestion

```

No need for docstring as we ensure the name is as explicit as it can be (plus GIVEN/WHEN/THEN which you already have)